parsl/channels/oauth_ssh/oauth_ssh.py

```diff
@@ -0,0 +1,140 @@
+import getpass
+import logging
+import select
+import paramiko
+import socket
+
+from parsl.errors import OptionalModuleMissing
+from parsl.channels.ssh.ssh import SSHChannel
+
+try:
+    from oauth_ssh.ssh_service import SSHService
+    from oauth_ssh.oauth_ssh_token import find_access_token
+    _oauth_ssh_enabled = True
+except (ImportError, NameError, FileNotFoundError):
+    _oauth_ssh_enabled = False
+
+
+logger = logging.getLogger(__name__)
+
+
+class OAuthSSHChannel(SSHChannel):
```

Comment:
UI-wise this might be better named GlobusSSHChannel, if that's really what this channel is for

Reply:
It is now oauth-ssh with code hosted here : https://github.com/xsede/oauth-ssh
There's some branding thing going on.